content/influxdb/v2.0/get-started.md

```diff
@@ -497,24 +520,19 @@ the `influx` command line interface (CLI).
 InfluxDB is now initialized with a primary user, organization, and bucket.
 You are ready to [write or collect data](/v2.0/write-data).
 
-{{% note %}}
-#### Use the influx CLI
+### (Optional) Set up and use the influx CLI
 
-To use the [`influx` CLI](/v2.0/reference/cli/influx) after setting up InfluxDB, provide your [authentication token](/v2.0/users/tokens/), which is automatically generated during the setup process. For instructions on viewing your token via CLI or UI, see [View tokens](/v2.0/security/tokens/view-tokens/).
+If you set up InfluxDB through the UI and want to use the [`influx` CLI](/v2.0/reference/cli/influx), we recommend setting up a configuration profile. This lets you avoid having to pass your InfluxDB [authentication token](/v2.0/users/tokens/) with each `influx` command. Complete the following steps to set up a configuration profile that stores your credentials.
 
-Use one of the following methods to provide your authentication token to the CLI:
+1. In a terminal, run the following command: 
+  ```sh
+  influx config create -n default -u $INFLUX_URL -o $INFLUX_ORG -t $INFLUX_TOKEN -a
```

Comment:
all the other command line examples use actual values like   
```
-u http://localhost:9999 \
  -t mySuP3rS3cr3tT0keN \
  -o example-org
```

instead of env variables. I'm not sure which is better, but they should probably all be consistent.

Reply:
Thank you @russorat - updated both sections